src/test/java/wooteco/subway/admin/acceptance/AcceptanceTest.java

```diff
@@ -163,12 +165,46 @@ void removeLineStation(Long lineId, Long stationId) {
         given().
                 contentType(MediaType.APPLICATION_JSON_VALUE).
                 accept(MediaType.APPLICATION_JSON_VALUE).
-                when().
+        when().
                 delete("/lines/" + lineId + "/stations/" + stationId).
-                then().
+        then().
                 log().all().
                 statusCode(HttpStatus.NO_CONTENT.value());
     }
 
+    void setSubwayInformation() {
+        LineResponse lineResponse2 = createLine("2호선");
+        LineResponse lineResponse7 = createLine("7호선");
+        LineResponse lineResponseB = createLine("분당선");
+
+        StationResponse stationResponse = createStation("왕십리");
```

Comment:
```suggestion
        StationResponse 왕십리 = createStation("왕십리");
```
이런식으로 바꾸면 테스트 코드도 좀더 이해하기 쉽지 않을까요?

Reply:
수정하였습니다.